docs/ivys.js

```diff
@@ -78,9 +78,9 @@ Product.prototype.removeTag = function () {
 var ProductCollection = function () {
     "use strict";
     var self = this;
-    self.all_products = {};
-    self.favorite_products = {};
-    self.all_tags = {};
-    self.selected_tags = {};
-    self.selected_products = {};
+    self.all_products = {}; // sku number : 'product name' ?
+    self.favorite_products = {}; // sku number : 'product name' ?
+    self.all_tags = {}; // 'tag-name' : 'tag-name' ?
```

Comment:
The products are in all_products... I don't get what you mean. Another object representing what product? Like just one?